### DISCUSSION TITLE
duplication between local and cloud storage

### DISCUSSION THREAD

**Discussion by Radvendii** [pixelfed/pixelfed#5283] [General]
We've configured pixelfed to use s3 storage. But we've observed that images are still getting saved to the local filesystem.

As we understand it, media is first uploaded to local storage and then sent to the cloud (s3). When loading an image right after it's uploaded, it seems to be served from local storage.

- Are we understanding this correctly?
- Is this temporary?
- When/how does it transition to being served from the cloud?

CC @Niols

**Comment by xhabih** [comment]
Same issue here.
